cli/command/image/pull.go

```diff
@@ -67,15 +69,18 @@ func runPull(cli command.Cli, opts pullOptions) error {
 	// Check if reference has a digest
 	_, isCanonical := distributionRef.(reference.Canonical)
 	if !opts.untrusted && !isCanonical {
-		err = trustedPull(ctx, cli, imgRefAndAuth, opts.platform)
+		err = trustedPull(ctx, cli, imgRefAndAuth, opts)
 	} else {
-		err = imagePullPrivileged(ctx, cli, imgRefAndAuth, opts.all, opts.platform)
+		err = imagePullPrivileged(ctx, cli, imgRefAndAuth, opts)
 	}
 	if err != nil {
 		if strings.Contains(err.Error(), "when fetching 'plugin'") {
 			return errors.New(err.Error() + " - Use `docker plugin install`")
 		}
 		return err
 	}
+	if opts.quiet {
+		fmt.Fprintln(cli.Out(), imgRefAndAuth.Reference().String())
```

Comment:
We should just do this even without the quiet flag.
WDYT @thaJeztah @vdemeester?

Reply:
Without quiet 
```
latest: Pulling from vdemeester/moby-dev-criu
c73ab1c6897b: Already exists 
1ab373b3deae: Already exists 
b542772b4177: Already exists 
57c8de432dbe: Already exists 
c81227e1ec90: Already exists 
c368dcc533d6: Already exists 
a0037d23043c: Already exists 
f64db6c96f10: Already exists 
6993efc377f4: Already exists 
963f9c7d7720: Pull complete 
Digest: sha256:9c5be10571181cd25ff13d9775c63017aeefc067df5eaa223375f9cf6b02d176
Status: Downloaded newer image for vdemeester/moby-dev-criu:latest
```

The reference already appears, in a different representation though so.. I wonder :stuck_out_tongue: